src/db/db-helper.js

```diff
@@ -241,22 +245,92 @@ module.exports = class DBHelper {
 
   static async updateEventStatusWithdrawing(currBlockTime) {
     try {
-      await db.Events.update(
+      const updatedEvents = await db.Events.update(
```

Comment:
return the updatedDocs and then pass it to the sync logic for updateLeaderboardWithdrawing